benchmark/child_process/child-process-exec-stdout.js

```diff
@@ -1,41 +1,42 @@
 'use strict';
 const common = require('../common.js');
+const { exec, execSync } = require('child_process');
+const isWindows = process.platform === 'win32';
 
 var messagesLength = [64, 256, 1024, 4096];
-// Windows does not support that long arguments
-if (process.platform !== 'win32')
-  messagesLength.push(32768);
-const bench = common.createBenchmark(main, {
+// Windows does not support arguments that long
+if (!isWindows) messagesLength.push(32768);
+
+const bench = common.createBenchmark(childProcessExecStdout, {
   len: messagesLength,
   dur: [5]
 });
 
-const child_process = require('child_process');
-const exec = child_process.exec;
-function main(conf) {
+function childProcessExecStdout(conf) {
   bench.start();
 
-  const dur = +conf.dur;
+  const maxDuration = conf.dur * 1000;
   const len = +conf.len;
 
-  const msg = `"${'.'.repeat(len)}"`;
-  // eslint-disable-next-line no-unescaped-regexp-dot
-  msg.match(/./);
-  const options = {'stdio': ['ignore', 'pipe', 'ignore']};
-  const child = exec(`yes ${msg}`, options);
+  const cmd = `yes "${'.'.repeat(len)}"`;
+  const child = exec(cmd, { 'stdio': ['ignore', 'pipe', 'ignore'] });
 
   var bytes = 0;
-  child.stdout.on('data', function(msg) {
+  child.stdout.on('data', (msg) => {
     bytes += msg.length;
   });
 
-  setTimeout(function() {
+  setTimeout(() => {
     bench.end(bytes);
-    if (process.platform === 'win32') {
-      // Sometimes there's a yes.exe process left hanging around on Windows...
-      child_process.execSync(`taskkill /f /t /pid ${child.pid}`);
+    if (isWindows) {
+      // Sometimes there's a yes.exe process left hanging around on Windows.
+      try {
```

Comment:
This `try` should get rid of [this error](https://ci.nodejs.org/job/node-test-binary-windows/8986/RUN_SUBSET=2,VS_VERSION=vs2015,label=win2012r2/console), right?